### PR TITLE
fix curve widget

### PR DESCRIPTION
**Pull request by nuria-balaguer**

### CODE REVIEW

File: src/LineChart/loadChart.js [hunk 1]
```diff
@@ -24,7 +30,20 @@ const loadChart = (chartData, elm, columns, options, onError) => {
...
+                ...chartData,
+                [
+                    new Date(),
+                    0,
+                    ...fillerNulls(columns),
+                ],
+            ]
+
+            console.log('columns', columns)
+            console.log('chartData', chartData)
```
Comment (dimaPortyanka) [inline src/LineChart/loadChart.js:43]: Please remove it

File: src/TemperatureChart2/TemperatureChart2.js [hunk 2]
```diff
@@ -94,22 +108,40 @@ const TemperatureChart2 = (props) => {
 
     const value = useMemo(() => {
         if (!sensorData) {
-            return []
+            const date = new Date()
+
+            return [[
+                new Date(date.getTime() - 24 * 60 * 60 * 1000),
```
Comment (dimaPortyanka) [inline src/TemperatureChart2/TemperatureChart2.js:114]: 24x60x60x1000 better if it would be As constant

File: src/LineChart/loadChart.js (hunk 1 above)
Comment (nuria-balaguer) [reply src/LineChart/loadChart.js]: fixed

File: src/TemperatureChart2/TemperatureChart2.js (hunk 2 above)
Comment (nuria-balaguer) [reply src/TemperatureChart2/TemperatureChart2.js]: fixed

File: src/TemperatureChart2/TemperatureChart2.js [hunk 3]
```diff
@@ -28,19 +28,25 @@ const propTypes = {
...
     onFullScreen: noop,
+    customColumns: [],
+    customData: [],
     isChartPrinting: false,
     userOptions: options,
     temperatureChartFullscreen: false,
     setTemperatureChartFullscreen: noop,
 }
 
+const DAY_BEFORE = 24 * 60 * 60 * 1000
```
Comment (dimaPortyanka) [inline src/TemperatureChart2/TemperatureChart2.js:48]: I would rather name it day

**Review by dimaPortyanka — APPROVED** [review]

**Review by AnastasiiaManilnikova — APPROVED** [review]

File: src/TemperatureChart2/TemperatureChart2.js (hunk 3 above)
Comment (nuria-balaguer) [reply src/TemperatureChart2/TemperatureChart2.js]: fixed

**Review by dimaPortyanka — APPROVED** [review]

**Review by AnastasiiaManilnikova — APPROVED** [review]

**Pull request merged**